crates/core/src/factories/core_space.rs

```diff
@@ -374,7 +375,12 @@ impl Space for CoreSpace {
                         }
 
                         // add it to bootstrapping.
-                        bootstrap.put(info);
+                        bootstrap.put(info.clone());
+
+                        // and send it to our peers.
+                        if let Err(err) = broadcast_agent_info(peer_store, local_agent_store, publish, info).await {
```

Comment:
Turns out we never hooked this up. We have the ability to broadcast agent info to our peers rather than waiting for them to bootstrap but we need to actually do that when updating our arc.